projects/core/src/lib/services/dialog.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+import { DialogType } from "../models/dialogType";
+
+@Injectable()
+export class DialogService {
+	public $isOpen: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+	public $dialogType: BehaviorSubject<DialogType> = new BehaviorSubject<DialogType>(
+		DialogType.NONE
+	);
+
+	public openDialog(dialogType: DialogType): void {
+		this.$isOpen.next(true);
+		this.$dialogType.next(dialogType);
+	}
+	public closeDialog(): void {
+		this.$isOpen.next(false);
+		this.$dialogType.next(DialogType.NONE);
+	}
```

Comment:
please add unit tests for openDialog and closeDialog methods

Reply:
done